File: 프로그래머스/1/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.py
def solution(t, p):
    answer = 0
    length = len(t)-len(p)+1 # 탐색해야 할 횟수
    
    for i in range(length):
        if int(t[i:i+len(p)]) <= int(p):
            answer += 1
    
    return answer

File: 프로그래머스/1/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.js
function solution(t, p) {
    var answer = 0;
    const length = t.length - p.length
    
    for (i = 0; i <= length; i++) {
        if (parseInt(t.substr(i, p.length)) <= parseInt(p)) {
            answer++;
        }
    }
    
    return answer;
}